.github/workflows/pytorch_gpu_tests.yml

```
@@ -0,0 +1,40 @@
+name: Pytorch Unit Tests (with GPU) 
+
+on:
+  pull_request:
+    branches:
+      - 'main'
```

```suggestion
      - 'main'
      - 'mq-working-branch**'
```

I think this is needed to run this GH action for merge queue